### PR TITLE
update to support new neo

### PR DESCRIPTION
**Pull request by lightszero**
fixed https://github.com/neo-project/neo-devpack-dotnet/issues/181

finish it.
can merge now.

And need help for fix the native unittest?I am not familiar with that.
@neo-project/core

### CODE REVIEW

**Comment by shargon** [comment]
I will help you with UT

File: src/Neo.SmartContract.Framework/OperandSizeAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Neo.VM
+{
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = false)]
+    internal class OperandSizeAttribute : Attribute
```
Comment (shargon) [inline src/Neo.SmartContract.Framework/OperandSizeAttribute.cs]: We can remove this class if we merge https://github.com/neo-project/neo-vm/pull/285

File: src/Neo.Compiler.MSIL/MSIL/Converter.cs [hunk 2]
```diff
@@ -679,15 +682,15 @@ private int ConvertCode(ILMethod method, OpCode src, NeoMethod to)
                 case CodeEx.Brtrue:
                 case CodeEx.Brtrue_S:
                     {
-                        var code = _Convert1by1(VM.OpCode.JMPIF, src, to, new byte[] { 0, 0 });
+                        var code = _Convert1by1(VM.OpCode.JMPIF_L, src, to, new byte[] { 0, 0, 0, 0 });
```
Comment (shargon) [inline src/Neo.Compiler.MSIL/MSIL/Converter.cs]: We should use `JMPIF` if the address is less than one byte

File: tests/Neo.SmartContract.Framework.UnitTests/Services/Neo/NativeTest.cs [hunk 3]
```diff
@@ -30,14 +32,14 @@ public void Init()
...
             using (var script = new ScriptBuilder())
             {
-                script.EmitSysCall(InteropService.Neo_Native_Deploy);
+                script.EmitSysCall(TestEngine.Native_Deploy);
                 _engine.LoadScript(script.ToArray());
-                _engine.Execute();
+                Assert.AreEqual(VMState.HALT, _engine.Execute());
             }
+
+            _engine.AddEntryScript("./TestClasses/Contract_Native.cs");
```
Comment (shargon) [inline tests/Neo.SmartContract.Framework.UnitTests/Services/Neo/NativeTest.cs]: @lightszero fixed the Deploy method, it's needed for the test, now the failure seems to be other error

File: src/Neo.SmartContract.Framework/OperandSizeAttribute.cs (hunk 1 above)
Comment (lightszero) [reply src/Neo.SmartContract.Framework/OperandSizeAttribute.cs]: we can do that on other pr

File: src/Neo.Compiler.MSIL/MSIL/Converter.cs (hunk 2 above)
Comment (lightszero) [reply src/Neo.Compiler.MSIL/MSIL/Converter.cs]: we can do that on other pr

Comment (lightszero) [reply src/Neo.Compiler.MSIL/MSIL/Converter.cs]: we can do that on other pr

**Pull request merged**